Control/Seq.hs

```diff
@@ -146,6 +148,11 @@ seqFoldable strat = seqList strat . toList
 
 {-# SPECIALISE seqFoldable :: Strategy a -> Strategy [a] #-}
 
+-- | Evaluate the elements of a bifoldable data structure according to
+-- the given strategy
+seqBifoldable :: Bifoldable p => Strategy a -> Strategy b -> Strategy (p a b)
+seqBifoldable = bifoldMap
```

Comment:
This doesn't work, the `Monoid ()` instance doesn't force anything.

This should work:
```suggestion
seqBifoldable strat1 strat2 = bifoldr (\x acc -> strat1 x `seq` acc) (\y acc -> strat2 y `seq` acc) ()
```